datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/scripts/__init__.py

```diff
@@ -5,8 +5,9 @@
 
 from ...console import CONTEXT_SETTINGS
 from .metrics2md import metrics2md
+from .upgrade_python import upgrade_python
 
-ALL_COMMANDS = (metrics2md,)
+ALL_COMMANDS = (metrics2md, upgrade_python)
```

Comment:
I don't feel like commands in the `scripts` group are very discoverable, unfortunately… So how about this target usage…?

```console
$ ddev meta python upgrade
```

Reply:
I thought about that but couldn't think of another future subcommand. Can you?

Reply:
Not really, but up to now we also had only the Markdown table tool under `scripts`, right? 😛 I guess I'm fine either way.